wfe/wfe.go

```diff
@@ -1010,6 +1010,10 @@ func (wfe *WebFrontEndImpl) prepChallengeForDisplay(request *http.Request, authz
 	if challenge.Error != nil && !strings.HasPrefix(string(challenge.Error.Type), probs.V1ErrorNS) {
 		challenge.Error.Type = probs.V1ErrorNS + challenge.Error.Type
 	}
+
+	if features.Enabled(features.ForceConsistentStatus) {
+		challenge.Status = authz.Status
```

Comment:
This will mark all challenges in an authorization as valid, not just the one that we did validate, which seems like it'll undoubtedly lead to confusion down the road.

Reply:
We have the option to remove the non-valid challenges from a valid authz, maybe that's a good solution to this confusion? (Though, probably just replacing it with a different "Where is challenge type X" confusion...)

Reply:
Alright, I'll narrow this so it only takes effect in the "invalid" case, which I think is the main one we are running into.